ada MoveHand to the current DOF values

When the 2 fingers are already in DOF values - [0.8,0.8], or close to [0.8,0.8], then if I call hand.MoveHand(0.8,0.8), it will raise an exception - TrajectoryExecutionFailed. Is it normal? 

``` python
[ERROR] [adapy:adarobot.py:228]:ExecuteTrajectory: Trajectory execution failed.
Traceback (most recent call last):
  File "/home/shenli/storage/ada/src/ada/adapy/src/adapy/adarobot.py", line 222, in ExecuteTrajectory
    traj_future.result(timeout)
  File "/home/shenli/storage/ada/src/ada/adapy/src/adapy/futures.py", line 70, in result
    raise self._exception
TrajectoryExecutionFailed: Trajectory execution failed (REJECTED): INVALID_GOAL
```

--- a/adapy/src/adapy/adarobot.py
+++ b/adapy/src/adapy/adarobot.py
@@ -198,6 +198,7 @@ class ADARobot(Robot):
             return Robot.ExecuteTrajectory(self, traj, defer=defer, timeout=timeout,
                                            **kwargs)
 
+
         # Check that the current configuration of the robot matches the
         # initial configuration specified by the trajectory.
         if not prpy.util.IsAtTrajectoryStart(self, traj):

